src/containers/Devices.jsx

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux'
```

Comment:
Well, the argument in favor of separate dumb component/view and its connected version is that it's easier to test.